Automattic/wp-calypso
# Migrate credit card form styles to Webpack

#### Changes proposed in this Pull Request

* Migrate credit card form styles to Webpack

<img width="696" alt="Screen Shot 2019-08-14 at 8 17 53 PM" src="https://user-images.githubusercontent.com/2036909/63064722-b0313280-bed0-11e9-954e-d99f7ed23cf4.png">


#### Testing instructions

* Sandbox the API so that you see the correct form fields (otherwise they will use different markup).
* Add a plan to your cart.
* Visit the checkout page.
* Verify that the credit card form fields (particularly the number, expiry, and cvv) look similar (see above screenshot).

Related to #35196 and #33620
